plugins/groq/src/index.ts

```diff
@@ -14,69 +14,63 @@
  * limitations under the License.
  */
 
-import { genkitPlugin, Plugin } from '@genkit-ai/core';
+// Import necessary types and functions for Groq SDK integration.
 import Groq from 'groq-sdk';
-import {
-  llama3x70b,
-  llama3x8b,
-  gemma7b,
-  mixtral8x7b,
-  groqModel,
-  SUPPORTED_GROQ_MODELS,
-} from './groq_models';
+import { groqModel, SUPPORTED_GROQ_MODELS } from './groq_models';
 
-export { llama3x70b, llama3x8b, gemma7b, mixtral8x7b };
+// Exporting SUPPORTED_GROQ_MODELS for external modules to access available models easily
+export { SUPPORTED_GROQ_MODELS };
```

Comment:
We shouldn't export this if the plugin didn't already

Reply:
Done